integrations/tensorflow/configure_iree.bzl

```diff
@@ -0,0 +1,36 @@
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "patch")
+
+def _is_absolute(path):
+    """Returns `True` if `path` is an absolute path.
+    Args:
+      path: A path (which is a string).
+    Returns:
+      `True` if `path` is an absolute path.
+    """
+    return path.startswith("/") or (len(path) > 2 and path[1] == ":")
+
+def _join_path(a, b):
+    if _is_absolute(b):
+        return b
+    return str(a) + "/" + str(b)
+
+def _configure_iree_impl(repository_ctx):
+    current_workspace_path = repository_ctx.path(
+        repository_ctx.attr.current_workspace,
+    ).dirname
+
+    full_path = _join_path(current_workspace_path, repository_ctx.attr.path)
+
+    repository_ctx.symlink(full_path, "")
+
+    patch(repository_ctx, patch_cmds = ["""grep -rl --exclude-dir=bazel-* "@mlir-hlo" | xargs sed -i 's|@org_tensorflow//tensorflow/compiler/mlir/hlo|@org_tensorflow//tensorflow/compiler/mlir/hlo|'"""])
```

Comment:
Mark my words: I will end up debugging why this incantations formats the hard drive of a windows machine.

Reply:
I'm trying to wrap my head around why you can only run the sed command on files that don't contain "@mlir-hlo".

Wdyt about flipping this:
  sed -i 's|...|' $(find . -name BUILD) $(find . -name BUILD.bazel)

Or at least take the -r off of your grep and qualify it with an explicit list of files ala my find commands above. I generally dislike a global search/replace for a keyword without some qualifications.

Also, is it just my eyes and a too long line or is this replacing a string with the same string?

Reply:
Also, when I said that I would fork the repo and run sed...

https://www.youtube.com/watch?v=QiZNSzWIaLo

Reply:
:grin: That's a recursive grep, not an excluding grep (`-r` not `-R`, duh!). But actually this whole rule is unnecessary and I forgot to delete the file.

But as a general rule, the only thing I've attempted so far with this PR is "this builds" not "this is at all understandable"